packages/frontend/src/components/ChartConfigPanel/index.tsx

```diff
@@ -1,23 +1,19 @@
 import { Button } from '@blueprintjs/core';
 import { Popover2 } from '@blueprintjs/popover2';
 import { ChartType } from '@lightdash/common';
-import React, { useState } from 'react';
+import React from 'react';
 import { useVisualizationContext } from '../LightdashVisualization/VisualizationProvider';
 import ChartConfigTabs from './ChartConfigTabs';
 
 export const ChartConfigPanel: React.FC = () => {
     const { chartType, plotData } = useVisualizationContext();
     const disabled = !plotData || chartType === ChartType.TABLE;
 
-    const [isOpen, setIsOpen] = useState(false);
     return (
         <Popover2
             content={<ChartConfigTabs />}
             interactionKind="click"
-            isOpen={isOpen}
```

Comment:
does it mean it is "always open" now ?

Reply:
No, it means we don't control the popover open state, it knows when to open and close.